src/compas_view2/app/app.py

```diff
@@ -53,7 +53,10 @@
 
 HERE = os.path.dirname(__file__)
 ICONS = os.path.join(HERE, "../icons")
-CONFIG = os.path.join(HERE, "config.json")
+
+with open(os.path.join(HERE, "config_default.json")) as f:
+    DEFAULT_CONFIG = json.load(f)
```

Comment:
This should be moved back into the init function of App. Generally file operations should not be put "naked" at the global scope of file.